README.md

```diff
@@ -86,26 +86,30 @@ pip uninstall jupyterlab_accessible_themes
     cd jupyterlab_accessible_themes
    ```
 
-3. Install the package in development mode
-
-   ```bash
-   pip install -e .
-   ```
-
-4. Now you'll need to link the development version of the extension to JupyterLab and rebuild the Typescript source:
+3. Install the node dependencies and build the extension
 
    ```bash
    # Install node dependencies
    jlpm install
 
    # Compile packages before linking to Jupyterlab development version
    jlpm build
+   ```
```

Comment:
As a new contributor to the project (and who has perhaps never worked on Lab) - how does one install `jlpm`?

The pre-requisites listed above do not help in any way with these instructions